src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -63,6 +63,14 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
 
             if (sourceType.IsTupleType)
             {
+                // TODO(https://github.com/dotnet/roslyn/issues/12389):
+                // Need to save/restore tupleness as well
+                if (sourceType.IsTupleType)
```

Comment:
Right, the bug mentioned has more details, but this is meant to short-circuit the code path since there are numerous bugs around the dynamic/tuple combinations right now that prevent round tripping tests from passing.